flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/WindowJoinUtil.scala

```diff
@@ -185,25 +186,22 @@ object WindowJoinUtil {
       if (
         leftWindowProperties.getTimeAttributeType != rightWindowProperties.getTimeAttributeType) {
         throw new TableException(
-          s"""
-             |Currently, time attribute type of left and right inputs should be both row-time or
-             |both proc-time. In the future, we could support different time attribute type.
-             |""".stripMargin)
+          "Currently, window join doesn't support different time attribute type of left and " +
+            "right inputs.\n" +
+            s"The left time attribute type is ${leftWindowProperties.getTimeAttributeType}.\n" +
+            s"The right time attribute type is ${rightWindowProperties.getTimeAttributeType}.")
       } else if (leftWindowProperties.getWindowSpec != rightWindowProperties.getWindowSpec) {
         throw new TableException(
-          s"""
-             |Currently, the windowing TVFs must be the same of left and right inputs.
-             |In the future, we could support different window TVFs, for example, tumbling windows
-             | join sliding windows with the same window size.
-             |""".stripMargin)
+          "Currently, window join doesn't support different window table function of left and " +
+            "right inputs.\n" +
+            s"The left window table function is ${leftWindowProperties}.\n" +
```

Comment:
We just need to print the windowSpec instead of the total `leftWindowProperties`.  Besides, we need to implement `toString` method for WindowSpecs.